### DISCUSSION TITLE
Finally some nicely laid out operating system code

### DISCUSSION THREAD

**Discussion by marinusmaurice** [opuntiaOS-Project/opuntiaOS#28] [General]
Firstly I haven't at this point pulled the repo, but i really felt the need to say that the code looks well laid out (well... in terms of the files at least). It doesn't have 100's of different file types requiring complex make or cmake scripts etc. So far. I like.

But I do have a few questions:
-Is the code based on some other code or is it all your own work?
-What is the licensing on it?

I hope it will build first time. I honestly am so so so frustrated at the state of .Net and its tools on linux and 9 out of every 10 repos I pull has problems. 

Anyways, I just felt the need to compliment on the clean and uniform layout of the source code. Its a real joy to see. Now.... lets hope it builds first time at least ;)

mvm

**Comment by marinusmaurice** [comment]
So.. its building. I just had a thought. Is this project perhaps based on the  linux from scratch project? 
Forgive me if I come across as a bit "green" in the linux world. I've been a long time admirer of the work done on the GNU/Linux projects and only recently fully transitioned from windows.

**Reply by tripleo1** [reply]
This project is not derived from Linux and has no Linux code in it.
